packages/experimental/Avatar/src/titles.ts

```diff
@@ -101,6 +101,7 @@ export const titles = new Set<string>([
   'The Rt Hon Sir',
   'The Rt Hon Visc',
   'Viscount',
```

Comment:
Wait so we actually have a hardcoded list of titles for use in GetInitials? I'm assuming we got this list from the web component?

Reply:
Web doesn't have it. Here's their implementation and they don't remove titles: https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-avatar/src/utils/getInitials.ts